### DISCUSSION TITLE
Visual Studio 2022 에서 PDcurses 설치

### DISCUSSION THREAD

**Discussion by hongsam14** [Kookmin-Univ-C-programming/SnakeGame#9] [General]
[ 교수님 링크](https://github.com/rbergen/PDCurses) 

[2017에서 pdcurses 설치하는 글](https://stackoverflow.com/questions/42708392/install-pdcurses-on-visual-studio-2017) 

아직 환경 구축 전인데 참고하려는 자료들 올려봅니다.

**Comment by mumat0103** [comment]
확인했습니다 저도 참고하겠습니다

**Comment by hongsam14** [comment]
설치랑 구동 확인했습니다
![screen](https://user-images.githubusercontent.com/69339846/166471071-e3b6d83f-75ee-446b-9f44-d5a4d668e14a.JPG)
.

**Comment by mumat0103** [comment]
저도 설치랑 구동 확인했습니다. 근데 윈도우에서 돌리려고 하니까 이것저것 할 게 많아서 wsl에서 ncurses  쓰는 게 더 편한거같네용..

**Reply by hongsam14** [reply]
맞아요...

**Reply by hongsam14** [reply]
visual studio 프로젝트랑 솔루션은 깃에 올려본 적이 없어서 아직은 올리지 않겠습니다. 설사 올리더라도 브랜치 파서 올릴께요
